docs/serving/samples/knative-routing-go/index.md

```diff
@@ -260,6 +260,39 @@ The Gateway proxy checks the updated host, and forwards it to `Search` or
 
 ![Object model](./images/knative-routing-sample-flow.png)
 
+## Using internal services and `"httpProtocol": "Redirected"`
+
+Using the above approach, services will be available using two entrypoints into the cluster:
+The original ones provided by KNative Serving (`search-service.default.example.com` and `login-service.default.example.com`),
+as well as the additional entrypoints `example.com/search` and `example.com/login`
+provided by the manually added VirtualService (`entry-route`).
+
+To make sure your service can only be reached via the manually created
+VirtualService, you can add the annotation `networking.knative.dev/visibility: cluster-local`
+to the KNative Service definitions, and route traffic over
+`knative-local-gateway.istio-system.svc.cluster.local` with a destination address of an internal service,
+instead of the public ingress one at `istio-ingressgateway.istio-system.svc.cluster.local`
+with a destination address of an externally available service.
+
+Have a look at
+
+```
+kubectl apply --filename docs/serving/samples/knative-routing-go/sample-internal.yaml
```

Comment:
Before applying this yaml, we need to replace the images in `sample-internal.yaml`  by `perl -pi -e "s@github.com/knative/docs/docs/serving/samples@${REPO}@g" docs/serving/samples/knative-routing-go/sample-internal.yaml` again?

I think we should either add `sample-internal.yaml` in `Setup` section above or to run command `kubectl label kservice search-service login-service networking.knative.dev/visibility=cluster-local` instead of `kubectl apply`

Reply:
I like the `kubectl label kservice search-service login-service networking.knative.dev/visibility=cluster-local` approach. Was not really happy with duplicating the file anyways.